### PR TITLE
Add duplicate prices feature

### LINKED ISSUE #14958: Duplicate Product in BO with different price for each shop (multishop)
**To Reproduce**

1. Set a multishop with 2 shops (or more). For instance:
2. Create a product in context all shop
3. On each shop set a different price for the product
4. Finally on products board, select the previous product and click on duplicate product on all shop context.

**Describe the bug**

A product duplicated from all shop context set the default shop's price for all shops. It should set the same price than the original product (the price corresponding to the shop).

**Additionnal information**
PrestaShop version: 1.7.x
PHP version: all

### PR DESCRIPTION
**Pull request by 202-ecommerce**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | add the duplication of a product price in multishop when the product has a different price in each shops
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #14958 .
| How to test?  | Steps to reproduce the behavior:

Set a multishop with 2 shops (or more). For instance:
Create a product in context all shop
On each shop set a different price for the product
Finally on products board, select the previous product and click on duplicate product on all shop context.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/14959)

### CODE REVIEW

File: classes/Product.php
Comment (matthieu-rolland) [inline classes/Product.php:4477]: Hello, thanx a lot for your contribution !

Could you please add a proper phpdoc including a short description of the method ?

**Review by matthieu-rolland — CHANGES_REQUESTED** [review]
Thank you for this PR, I made a minor comment, could you check it out?

**Comment by 202-ecommerce** [comment]
Hi @matthieu-rolland ,
I just added the PHPDoc as you asked and validated the PrettyCI

**Review by matthieu-rolland — DISMISSED** [review]

**Comment by matthieu-rolland** [comment]
thanx @202-ecommerce ! your fix will be tested by the QA before we can merge it.

**Comment by khouloudbelguith** [comment]
Hi @202-ecommerce,

The `Price per unit (tax excl.) ` is not duplicated, if we duplicate a product, it remains to `0`.
image
Thanks!

**Comment by 202-ecommerce** [comment]
Hi @khouloudbelguith ,
Just made the fix for the Price per unit

**Comment by khouloudbelguith** [comment]
@202-ecommerce, thanks!
Waiting for a review.

**Review by matthieu-rolland — APPROVED** [review]
LGTM

**Pull request merged**
